pkg/driftctl.go

```diff
@@ -149,19 +149,33 @@ func (d DriftCTL) Run() (*analyser.Analysis, error) {
 		}
 	}
 
-	analysis, err := d.analyzer.Analyze(remoteResources, resourcesFromState)
+	analysis := d.analyzer.CompareEnumeration(remoteResources, resourcesFromState)
 	if err != nil {
```

Comment:
seems like you removed the error returning here

Reply:
Yes because it was never used, there is no error to return here, should we keep it anyway ?

Reply:
no there is no problem with that but you left the error check on the next line ?